Package.swift

```diff
@@ -0,0 +1,63 @@
+// swift-tools-version:5.0
+import PackageDescription
+
+let package = Package(
+    name: "Bugsnag",
+    platforms: [
+        .macOS(.v10_11),
+        .tvOS("9.2"),
+        .iOS("9.3"),
+    ],
+    products: [
+        .library(name: "Bugsnag", targets: ["Bugsnag"]),
+    ],
+    dependencies: [],
+    targets: [
+        .target(
+            name: "Bugsnag",
+            dependencies: [],
+            path: "Bugsnag",
+            sources: [
+                ".",
+                "Breadcrumbs",
+                "Client",
+                "Configuration",
+                "Delivery",
+                "Helpers",
+                "KSCrash",
+                "Metadata",
+                "Payload",
+                "Plugins",
+                "KSCrash/Source/KSCrash/Reporting/Filters",
+                "KSCrash/Source/KSCrash/Recording/Tools",
+                "KSCrash/Source/KSCrash/Recording",
+                "KSCrash/Source/KSCrash/Recording/Sentry",
```

Comment:
Can you check whether this list is necessary? It's missing `Storage` currently, which suggests it might not be.

Reply:
Removed in 5537bb87dc686fb920dc7fbd4a397e7dec186535